ckanext/datagovtheme/templates/snippets/sort_by.html

```diff
@@ -8,15 +8,15 @@
   {% snippet 'snippets/sort_by.html', sort=c.sort_by_selected %}
 
 #}
-<span class="form-select {{ 'form-group' }} control-order-by">
+<div class="form-group control-order-by">
```

Comment:
what's the template tags for? `{{ 'form-group'}}`

also, looks like we deleted `form-select`??